Formula/c/clinfo.rb

```diff
@@ -32,6 +32,12 @@ def install
   end
 
   test do
-    assert_match(/Device Type +[CG]PU/, shell_output(bin/"clinfo"))
+    output = shell_output(bin/"clinfo")
+    # OpenCL does not work on ephemeral ARM CI.
+    if Hardware::CPU.arm? && OS.mac? && ENV["HOMEBREW_GITHUB_ACTIONS"]
```

Comment:
Maybe worth adding a `Hardware.virtualized?` that checks `kern.hv_vmm_present`

Reply:
(This doesn't actually exist yet - was more of a suggestion of something to add to `brew` instead of the horrible HOMEBREW_GITHUB_ACTIONS hack)